fitlins/base.py

```diff
@@ -152,13 +154,14 @@ def second_level(analysis, block, space, deriv_dir):
             paradigm = pd.DataFrame(cols)
 
             fmri_glm.fit(data, design_matrix=paradigm)
-            stat_type = [c['type'] for c in block.contrasts if c['name'] == contrast][0]
+            stat_type = [c['type'] for c in block.contrasts if c['name'] == contrast] or ['T']
+            stat_type = stat_type[0]
```

Comment:
What's the case that's being considered here? Isn't `'type'` required?

Reply:
I'm trying to rack my brain, but for some reason I recall it wasn't? and T should be default? But maybe I'm wrong, so I'll check the spec.